src/kbmod/search/common.h

```diff
@@ -43,45 +41,34 @@ namespace search {
 
     // I can't believe string::format is not a thing until C++ 20
     const std::string to_string() const {
-      return "lh: " + std::to_string(lh) +
-        " flux: " + std::to_string(flux) +
-        " x: " + std::to_string(x) +
-        " y: " + std::to_string(y) +
-        " vx: " + std::to_string(vx) +
-        " vy: " + std::to_string(vy) +
-        " obs_count: " + std::to_string(obs_count);
+        return "lh: " + std::to_string(lh) + " flux: " + std::to_string(flux) + " x: " + std::to_string(x) +
```

Comment:
This was a little easier to read in the original formatting (one field per line). Dunno about disabling code formatting for this block, though.